src/handler.ts

```diff
@@ -6,39 +6,48 @@ import { sendMCProhibition } from './eventbridge/Send';
 import logger from './observability/Logger';
 import { MCRequest } from './utils/MCRequest';
 
-const { NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE } = process.env;
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE, SEND_TO_SMC } = process.env;
 
 logger.debug(
-  `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n\n`,
+  `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n 
+  Send to smc: '${SEND_TO_SMC}\n`,
 );
 
 const handler = async (
   event: DynamoDBStreamEvent,
   _context: Context,
   callback: Callback,
 ) => {
-  try {
-    logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
+  if (
+    process.env.SEND_TO_SMC != undefined &&
```

Comment:
Is there no way to use the SEND_TO_SMC variable directly that was derived on line 9?

Reply:
Yes, that's a shout